src/controllers/techniciansController.js

```diff
@@ -17,8 +18,24 @@ const getTechniciansBy = (name, lastName) => {
     return technician;
 };
 
+const removeTechniciansBy = id => {
+    const search = technicians.find(technician => technician.id.toString() === id);
+    const updatedTechnicians = technicians.filter(technician => !(technician.id.toString() === id));
+    if (search){
+        fs.writeFileSync(
+            __dirname + "/../data/technicians.json",
+            JSON.stringify(updatedTechnicians),
+            { encoding: 'utf8', flag: 'w' }
+        );
+        return ("Technician with id " + id + " deleted.").toString();
+    } else {
+        return ("Technician with id " + id + " not found.").toString();
```

Comment:
The same that above.
the toString() is not necessary. You can do:
return `Technician with id ${id} not found.`;
```suggestion
        return `Technician with id ${id} not found.`;
```